vision-component-archetype/src/main/resources/archetype-resources/__rootArtifactId__-build/pom.xml

```diff
@@ -57,8 +57,8 @@
                     <moduleName>${project.parent.name}</moduleName>
                     <moduleDescription>${project.description}</moduleDescription>
                     <moduleVersion>${version}</moduleVersion>
-                    <requiredIgnitionVersion>7.7.0</requiredIgnitionVersion>
-                    <requiredFrameworkVersion>6</requiredFrameworkVersion>
+                    <requiredIgnitionVersion>7.8.0</requiredIgnitionVersion>
+                    <requiredFrameworkVersion>7</requiredFrameworkVersion>
```

Comment:
Let's not generate `requiredFrameworkVersion` any more, since it's no longer used